telemetry/definitions/commonDefinitions.json

```diff
@@ -1423,6 +1425,15 @@
             ],
             "description": "User-selected time range type while starting an insights query"
         },
+        {
+            "name": "interactionType",
+            "type": "string",
+            "description": "User can click on thumbs up / down or comment or give rating to share feedback.",
+            "allowedValues": [
```

Comment:
why not add the values from `cwsprChatInteractionType` ?  the intention of this new, non-cwspr-specific metric is to replace the old one.

then please update the description for `cwsprChatInteractionType` like this:

            "description": "(Deprecated, use interactionType instead.) User interaction type, especially for AI-related features."